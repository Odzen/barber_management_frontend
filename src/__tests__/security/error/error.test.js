// import { render, screen, cleanup, fireEvent } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ErrorView from '../../../security/views/error'

describe('testing to ErrorView', () => {
  test('should return ErrorView Component', () => {
    let error = renderer.create(<ErrorView />).toJSON()

    expect(error).toMatchSnapshot()
  })
})
