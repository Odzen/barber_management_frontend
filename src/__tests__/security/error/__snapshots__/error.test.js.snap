// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing to ErrorView should return ErrorView Component 1`] = `
<div
  className="error404"
>
  <h1
    className="font"
  >
    404 NOT FOUND
  </h1>
  <img
    alt=""
    src="../../../assets/images/error404.png"
  />
  <h2>
    No hemos podido encontrar la p√°gina que buscas.
  </h2>
  <h4>
    Atentamente, 
  </h4>
  <h5>
    Engineers
  </h5>
</div>
`
